xcell/tests/test_cls.py

```diff
@@ -33,9 +37,11 @@ def get_config(fsky=0.2, fsky2=0.3):
         'baryons_power_spectrum': 'nobaryons',
     }
     dummy0 = {'mask_name': 'mask_dummy0', 'mapper_class': 'MapperDummy',
-              'cosmo': cosmo, 'nside': nside, 'fsky': fsky, 'seed': 0}
+              'cosmo': cosmo, 'nside': nside, 'fsky': fsky, 'seed': 0,
+              'dtype': dtype}
     dummy1 = {'mask_name': 'mask_dummy1', 'mapper_class': 'MapperDummy',
-              'cosmo': cosmo, 'nside': nside, 'fsky': fsky2, 'seed': 100}
+              'cosmo': cosmo, 'nside': nside, 'fsky': fsky2, 'seed': 100,
+              'dtype': dtype2}
```

Comment:
After getting confused lower in this file, I'd rename `dtype` and `dtype2` to `dtype0` and `dtype1` as the dummy mappers.

Reply:
Done